### PR TITLE
valid

### PR DESCRIPTION
**Pull request by Mikechavo**

### CODE REVIEW

File: spec/models/fictional_char_spec.rb [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+            age: 47,
+            enjoys: 'Playing pool and wearing a Kangol hat and impressing chicks') 
+            expect(fictional_char.errors[:image]).to_not be_empty
+              end   
+    it "should validate enjoys is greater than 10 characters long" do
+        fictional_char = FictionalChar.create(name: 'Date Mike',
+        age: 47, enjoys: 'Playing',
+        image: 'https://i.redd.it/voer03gdaj911.png') 
+        expect(fictional_char.errors[:enjoys]).to_not be_empty
+          end
```
Comment (elysemontano) [inline spec/models/fictional_char_spec.rb:35]: Double check your indentation in this file.

File: spec/requests/fictional_chars_spec.rb [hunk 2]
```diff
@@ -37,4 +39,67 @@
...
+                   age: 47,
+                   enjoys: 'Playing pool and wearing a Kangol hat and impressing chicks'
+                     }
+                    }
+                    post '/fictional_chars', params: fictional_char_params
+                  expect(response.status).to eq 422
+                  fictional_char = JSON.parse(response.body)
+                  expect(fictional_char['image']).to include "can't be blank"
+                  end
+                end
```
Comment (elysemontano) [inline spec/requests/fictional_chars_spec.rb:104]: Check indentation here as well.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Your logic looks great, it looks like indentation is a little out of place on the spec files.

**Comment by Mikechavo** [comment]
Fixed, will add in a new branch with the fixes.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**